src/components/FarmDualCard/FarmDualCard.tsx

```diff
@@ -78,23 +78,19 @@ const FarmDualCard: React.FC<{
     stakedAmounts?.totalStakedBase,
   );
 
-  const poolRateA = getRewardRate(stakingInfo.totalRewardRateA);
-  const poolRateB = getRewardRate(stakingInfo.totalRewardRateB);
-
-  const earnedUSD =
-    Number(stakingInfo.earnedAmountA.toSignificant()) *
-      dQuicktoQuick *
-      stakingInfo.quickPrice +
-    Number(stakingInfo.earnedAmountB.toSignificant()) *
-      Number(stakingInfo.rewardTokenBPrice);
+  const poolRateA = getRewardRate(
+    stakingInfo.totalRewardRateA,
+    stakingInfo.rewardTokenA,
+  );
+  const poolRateB = getRewardRate(
+    stakingInfo.totalRewardRateB,
+    stakingInfo.rewardTokenB,
+  );
 
-  const earnedUSDStr =
-    earnedUSD < 0.001 && earnedUSD > 0
-      ? '< $0.001'
-      : '$' + earnedUSD.toLocaleString();
+  const earnedUSDStr = getEarnedUSDDualFarm(stakingInfo);
 
   const rewards =
-    stakingInfo?.rateA * stakingInfo?.quickPrice +
+    stakingInfo?.rateA * stakingInfo?.dQuickPrice +
```

Comment:
This is still wrong.  We can't make the assumption that rateA will always be quick. 
We should either rename the staking info property to be quickRate. 
OR better yet, have a function which will resolve any token and convert it to USD value.